frontend/src/features/properties/components/forms/subforms/BuildingForm.tsx

```diff
@@ -252,8 +252,4 @@ const BuildingForm = (props: BuildingProps & FormikProps<any>) => {
   );
 };
 
-export default React.memo(BuildingForm, (prevProps, currentProps) => {
-  const prev = { formikProps: prevProps, field: prevProps.nameSpace };
-  const curr = { formikProps: currentProps, field: currentProps.nameSpace };
-  return formikFieldMemo(prev, curr) && prevProps.disabled !== currentProps.disabled;
-});
```

Comment:
This was stopping the props from changing when you clicked Edit on the Building page, and disabled was not being reset so the fields were uneditable